boilerplate/openshift/golang-osd-operator/operator-sdk-generate.sh

```diff
@@ -20,10 +20,6 @@ VER=$(osdk_version $OSDK)
 # This explicitly lists the versions we know about. We don't support
 # anything outside of that.
 case $VER in
-  'v0.8.2')
-      $OSDK generate openapi
-      $OSDK generate k8s
-      ;;
```

Comment:
I don't think I agree with removing this support. Deprecating v0.8.2 and leaving it untested is one thing. Removing this entirely will slow adoption by gcp-project-operator. Can we leave it in until they've upgraded?